src/utils/environment.ts

```diff
@@ -1,14 +1,20 @@
 class Environment {
+	hostname: string;
+
+	constructor() {
+		this.hostname = window.location.hostname;
+	}
+
 	get isLocal() {
-		return process.env.REACT_APP_DEPLOYMENT_ENV === 'local';
+		return this.hostname.includes('localhost');
```

Comment:
Jeg ville brukt Vite sine Modes i stedet, da det trolig er mer robust (fort gjort at en ingress endres og så glemmer vi å titte i denne filen).
For at foreslått endring her skal fungere må man sende inn `--mode local` ved bygging, f.eks. ved at man legger til en `"build:local": tsc && vite build --mode local`. Står litt mer om modes her: [Env Variables and Modes | Vite](https://vitejs.dev/guide/env-and-mode#node-env-and-modes).

```suggestion
		return import.meta.env.MODE === "local";
```

Reply:
Det er ikke lov til å bruke local som mode: `Error: "local" cannot be used as a mode name because it conflicts with the .local postfix for .env files.`, men `lokal` funker jo 🤷
Men ser ikke poenget med å lage en `build:local`, siden vi lokalt bare kjører opp en dev-server?

Reply:
Det var jeg ikke klar over! 🤯 `build:local` var jo i så fall for å kunne sende inn en annen Mode-verdi 🤔

Reply:
Men det gikk fint å kjøre `vite --mode lokal`, altså lokal kjøring!
Så per nå er tre moduser i bruk 👌